Development/nmos/components.h

```diff
@@ -8,6 +8,7 @@ namespace nmos
 {
     // Components (used in raw video flows)
```

Comment:
```suggestion
    // Components (for raw video flows since IS-04 v1.1, extended to coded video Flows since v1.3 by the entry in the Flow Attributes register)
```